src/watcher.js

```diff
@@ -71,16 +74,20 @@ function processEvents(events) {
   }
 }
 
+async function getLastBlockToProcess() {
+  const lastBlockNumber = await getBlockNumber(web3Instance)
+  return lastBlockNumber - requiredBlockConfirmations + 1
```

Comment:
Is this formula correct? Imagine that we have configured `0` confirmations which means that the bridge will handle an event as soon as a transaction was included in the block. In this case the formula will get `lastBlockToProcess` as the latest block number plus `1` - this means that `getPastEvents` needs to check events from the block which does not exist yet. Please correct me if I am wrong.

Reply:
Thanks for pointing this out @akolotov. I assumed that the minimum value for `requiredBlockConfirmations` would be `1` (just the block in which the Tx was mined). But if it can be `0` then I need to fix the formula. Maybe @rstormsf can clarify this

Reply:
Alex is right. Someone can say requiredblock is 0